lib/address-edit/AddressEdit.js

```
@@ -206,26 +196,30 @@ var stdin_default = (0, import_vue2.defineComponent)({
       setAreaCode,
       setAddressDetail
     });
-    (0, import_vue2.watch)(() => props.areaList, () => setAreaCode(data.areaCode));
-    (0, import_vue2.watch)(() => props.addressInfo, (value) => {
+    (0, import_vue.watch)(() => props.addressInfo, (value) => {
       (0, import_utils.extend)(data, DEFAULT_DATA, value);
-      setAreaCode(value.areaCode);
+      (0, import_vue.nextTick)(() => {
+        var _a;
+        const options = (_a = areaRef.value) == null ? void 0 : _a.getSelectedOptions();
+        if (options && options.every((option) => option && option.value !== import_utils2.AREA_EMPTY_CODE)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)